### ISSUE TITLE
Install hangs

### ISSUE THREAD

**Issue by wickley** [pyrocms/pyrocms#3693]
Using Homestead 0.4.0. The installer accepts the database information and user credentials. After clicking the Install button it goes to a screen that shows "Installing...". Browser spins for less than 20 seconds and stops. The "Installing..." continues to show indefinitely with nothing happening besides that message. It's obvious that the script has stopped with no warnings or errors.

**Comment by wickley** [comment]
Looked at the source of the install page and manually ran /installer/run/0 -  /installer/run/26 with all returning true. Guessing the installer has an issue with the javascript/ajax.

Going to the root domain returns NotFoundHttpException in RouteCollection.php line 161. Going to /admin redirects to /admin/login and displays the login page. Unfortunately, using the credentials that were set in the installer, and verified in the .env file returns "Incorrect email or password."

Huge fan of Pyro, but every alpha and beta I've tested has been unusable.

**Comment by RyanThompson** [comment]
Ya the system runs the seeders after the installer steps. Seeders are where you get your general data for install.

What do your Laravel error logs say?


Ryan Thompson
President / CEO / Streams Evangelist
PyroCMS, Inc.
www.pyrocms.com

t (877) 705-0806
m (309) 752-8581

**Comment by RyanThompson** [comment]
Also what is your local domain you're using? I think there are some issues when running it off a directory.


Ryan Thompson
President / CEO / Streams Evangelist
PyroCMS, Inc.
www.pyrocms.com

t (877) 705-0806
m (309) 752-8581

**Comment by wickley** [comment]
storage > logs has one file for today laravel-2016-04-10.log and shows one error related to when I changed the name of ADMIN_THEME to make sure the .env file was being pulled in correctly after being unable to login with the credentials in the .env file.

local.ERROR: InvalidArgumentException: View [login] not found.

There's nothing else in that log.

I setup a development domain on the Homestead VM using pyro.test

I ran php artisan db:seed but didn't notice any difference. I'm assuming that's not the proper seed command for the Pyro installation.

I'm going to try to install on a live web server and real domain to see what happens.

Thank you

**Comment by RyanThompson** [comment]
I would also check your PHP log files. It's likely something like permissions if I had to guess.


Ryan Thompson
President / CEO / Streams Evangelist
PyroCMS, Inc.
www.pyrocms.com

t (877) 705-0806
m (309) 752-8581
